client/src/components/dashboard/Preferences.js

```diff
@@ -267,132 +248,225 @@ class Preferences extends React.Component {
       updateUserPartial(_id, section, user[section]).then(res => {
         const { updatedUser } = res.data;
         this.props.saveUser(updatedUser);
-        // open "Saved" popup
+        // open "saved" popup
         this.setState({ [e.target.id]: true });
         // close popup 1 second later
         setTimeout( _ => this.resetPopUp(e.target.id), 1200);
```

Comment:
I remember this syntax showing up some other time and we talked a little about it... I'm not sure where it became common to write a timeout like this? But if I'm not wrong the `_` is an argument to the function now. In my view it would be better to just write `setTimeout(() => ... ), delay)`. But maybe `_` is like some standard, idk.